### DISCUSSION TITLE
Bring back the list of repositories on an organization's dashboard

### DISCUSSION THREAD

**Discussion by rdhelms** [community/community#11371] [Other Feature Feedback, Questions, & Ideas]
It seems that the left sidebar's list of repositories was recently replaced with a "View Organization" button and a "Browse Organization's Repositories" button. Personally, I really dislike viewing the repositories from the organization's view, because each repository's card in the table contains so much more information than I need, and it often requires scrolling to find the repository I need. The list of repositories on the dashboard was easy to skim, and it was my primary way of quickly accessing repos, so I have no idea why it would have been removed. I also really liked being able to glance at the activity while going to the repository.

![image](https://user-images.githubusercontent.com/7010586/153595710-bf36bd3c-f4c4-43af-86d1-ac96a41ff4c2.png)

My flow often looked like this:
- Push a commit to a repo
- Go to the org's dashboard
- Confirm that the activity shows what I just pushed
- Click on a repo in the sidebar (sometimes the one I just pushed to, but sometimes a different one)

But now there's an extra click to first go to the organization and then click on the repo that I need.

**Comment by markets** [comment]
100% agree! I also submitted the same feedback in https://github.com/github/feedback/discussions/11541

**Comment by jradick** [comment]
I agree, I don't like the new layout.  It looks nice but it's less useful.  I suppose it would be OK on a small screen with limited space but I'm never going to use this on a phone or a tablet or a laptop with a tiny screen.

**Comment by DavidACameron-USDA** [comment]
> Personally, I really dislike viewing the repositories from the organization's view, because each repository's card in the table contains so much more information than I need, and it often requires scrolling to find the repository I need. The list of repositories on the dashboard was easy to skim, and it was my primary way of quickly accessing repos, so I have no idea why it would have been removed.

I'd like to echo this sentiment.  The list of repos on the org dashboard was essentially the only reason why I used the dashboard page.  It offered a thin, quick way to skim the list of repos and jump to any one of them.  Now I'm forced to use the repositories page which contains way, WAY too much unless information for someone who just wants to get where they want to go.  It's slower because you have to scroll much further down the page and occasionally search or paginate through results.  I looked for a setting that removed all the extra garbage info from the repo list in order to shrink its size, but didn't find one.

Ultimately, this change has had a negative impact on my productivity.  If you're going to take away the list of repos on the dashboard, then we need a better or customizable alternative.

**Comment by canberkdurmus** [comment]
I am having a big negative impact of this change on my productivity. When I am working on multiple projects in my organization (almost always) I keep my code on my primary display and Dashboard on the secondary one. This setup has only two points, the first one is to be able to see the latest activity from other people and the second and the most important one is the accessibility of the repositories. Sorted by the latest activity, this was the best, and the search bar to filter all repositories easily. The important point of these two points is gone now.

Now I can see two buttons, one of them sends me to a page bloated with the information about each individual repositories and the other one is basically the same page with this one in addition to pinned repositories and people summary. They also have a weird design, whenever I look at them, I am thinking something is broken with the UI at first, then I remember.

Why the old side panel was useful:
- I could see the list of repositories with only names (not the extra info, I wrote that info, I don't need to see it in order to navigate between repos, I only need names)
- Sorted by the latest activity: if I am actively working on three or four projects I can see them on top together.
- Search bar: If I cannot see any repo just simply enter the beginning of the name of the repo and it's there.

Why the new side panel is useless:
- The pages that the new buttons send me almost basically have the same content
- I can see 4-5 repositories at max without scrolling there and the page is full of descriptions that I wrote for other people (not for me obviously)
- The only point of the Dashboard page is now people's activity but this is not a social network platform and this is way too less functionality for one whole page.

If you found the old side panel useless, why it is not removed from the personal dashboard but removed from the organization dashboard? If the new one is useful for you, why don't you change the personal dashboard side panel with the newer one?

I think the main reason that it's removed can be making people use the new command palette (cmd/ctrl K) for navigating between repositories. I frankly liked the command palette and I am planning to use it actively but you don't need to decrease the functionality of another place to promote a new solution.

I have been following multiple discussions about this change since the minute that the changes went live and I am not very hopeful it will be changed back or into something useful again. I am probably going to abandon the Dashboard.

Thank you for your time reading my feedback.

**Comment by Tony-Mc** [comment]
My use case was similar to others. However, the most painful part is that there's no quick access to recently updated repositories.
When working with many repositories, I see a need to quickly find the repo I just pushed to, to be able to create a PR.

My workflow used to be:
1. Keep organization dashboard open in one tab.
2. Push changes to one or more repos.
3. Go to the dashboard and open the repo(s).
4. Use the "PR from the new branch" prompt to create the PR(s).

I wouldn't really mind not having the repo list that much if I would get these PR prompts at the top of organization dashboard or above "Recent activity" in the org dashboard.

**Comment by salehmdev** [comment]
Completely agree with this discussion and would love to have that feature back. That was a handy list to access the repo I last pushed to. Hence why I looked up whether or not other people also were wondering about that feature returning.

In case anyone might find it useful, here are 2 solutions that worked for me, which almost filled that void:
1. Using the organization repository page, sorted by "Last updated".
2. Personal fav: using the "Ctrl+K" shortcut to directly jump to a repo using the keyboard, from anywhere on Github.
2.1. The modal that pops up also has Repositories listed toward the bottom.

**Comment by brian-mcnamara** [comment]
Just found this after posting the same issue. I pointed out the documentation has not been updated https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-your-organization-dashboard . Hoping the documentation can stay the way it is and get this feature back 🙏
